forge-gui/res/tokenscripts/c_4_4_a_robot_noblock.txt

```diff
@@ -0,0 +1,6 @@
+Name:Robot Token
+ManaCost:no cost
+Types:Artifact Creature Robot
+PT:4/4
+K:CARDNAME can't block.
```

Comment:
do this like #1551 
https://github.com/Card-Forge/forge/pull/1551/files#diff-694431d534d1d47f75f2289bb6c5bb6b563284c2be8247b07c9b44eab64992ef